### PR TITLE
Added missing WebsocketClientWorker.ipynb 

### PR DESCRIPTION
**Pull request by codebecker**
The notebook "WebsocketServerWorker" states to be a two notebook tutorial. The seccond notebook "WebsocketClientWorker" was missing. I copied it from the official colab of the [opend minded homepage](https://www.openmined.org/).

Also I added a minor fix to the code snippet in the notebook.

### CODE REVIEW

File: examples/tutorials/WebsocketClientWorker.ipynb [hunk 1]
```diff
@@ -0,0 +1,762 @@
...
+        "#module_path = '/content/PySyft'  # You want './PySyft' to be on your sys.path\n",
+        "#if module_path not in sys.path:\n",
+        "#    sys.path.append(module_path)"
+      ],
+      "execution_count": 0,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "fatal: destination path 'PySyft' already exists and is not an empty directory.\n",
```
Comment (LaRiffle) [inline examples/tutorials/WebsocketClientWorker.ipynb]: Can you clean the output?

File: examples/tutorials/WebsocketClientWorker.ipynb [hunk 2]
```diff
@@ -0,0 +1,762 @@
...
+      ],
+      "execution_count": 0,
+      "outputs": [
+        {
+          "output_type": "error",
+          "ename": "ModuleNotFoundError",
+          "evalue": "ignored",
+          "traceback": [
+            "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
+            "\u001b[0;31mModuleNotFoundError\u001b[0m                       Traceback (most recent call last)",
```
Comment (LaRiffle) [inline examples/tutorials/WebsocketClientWorker.ipynb]: You have an error in the output here too

File: examples/tutorials/WebsocketClientWorker.ipynb (hunk 1 above)
Comment (codebecker) [reply examples/tutorials/WebsocketClientWorker.ipynb]: I will clean the output but it is [copied straight from this example from openmined.org ](https://colab.research.google.com/drive/1Je1rk7olA9uTWWaqvvt4_gXf7yX1rTBm) which also includes this lines. May the example there should also be fixed.

Comment (codebecker) [reply examples/tutorials/WebsocketClientWorker.ipynb]: Or is this output produced as soon as i open it with my colab?

Comment (codebecker) [reply examples/tutorials/WebsocketClientWorker.ipynb]: I have cleard the ouputs and also moved the Websocket Tutorials to a own folder

**Pull request merged**